### DISCUSSION TITLE
Im Sorry SuperTux Fandom. I'm not right in my head😓

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2843] [General]
I don't know if I can post this because of my behaviors, but I want to give a message of apology that most likely won't accept me. I try not to steal assets, but I can't control myself, my brain tells me to do it and I listen to it. I want to apologize to everyone, especially RustyBox. I know grabbing your backup was wrong, but I couldn't control myself well. I do the best I can, but nothing, my mind always says *grab those assets, nothing is going to happen* and I listen to it. I'll admit it, I'm trash in person, I can't control myself in what I do, I try my best and I can't control it. I admit that I ruined the community of the game, I just didn't want to hurt them at all, it's just that I try but I can't do it, my head forces me to do it. I swear I'm trying to change something, but it doesn't work at all. I'm literally losing my mind, I don't know what to do, I try everything and it's the same, I accept my consequences, I'm sorry to everyone for what I've done. Well. I hope they accept my apology

**Comment by Rusty-Box** [comment]
I'm gonna be honest with you, I do not believe you unless I see actual attempts of change on your end as this happened many time by now. Not to mention that you now have re-uploaded art I made to your Instagram, that is not part of this open source project, thus not simply free to use, considering you do not own the characters/people involved nor their likeness, thus could have asked for permission, which was not given by neither me, Frost or Orange. So if you really mean what you have wrote here than I will kindly ask you to remove the image in question: the **El Orango** picture!

**Comment by Rusty-Box** [comment]
![grafik](https://github.com/SuperTux/supertux/assets/20615048/2b340fe0-71d0-438e-aaac-38941c93af66)

Speaking of not believing you. What am I looking at, huh? You think I'm dumb or something?

**Reply by VincentAraneo** [reply]
That was an attempt to re-upload it, but it wouldn't let me upload it because it had more than 100 files or that it was too heavy, so I'll leave it empty with nothing

**Comment by VincentAraneo** [comment]
By the way, I already removed this from the image of El Orango,

**Reply by Rusty-Box** [reply]
Thank you
